[2025-08-28T06:50:17.120+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-28T06:50:17.279+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_pipeline.load_data scheduled__2025-08-27T00:00:00+00:00 [queued]>
[2025-08-28T06:50:17.301+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_pipeline.load_data scheduled__2025-08-27T00:00:00+00:00 [queued]>
[2025-08-28T06:50:17.302+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 4
[2025-08-28T06:50:17.338+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data> on 2025-08-27 00:00:00+00:00
[2025-08-28T06:50:17.348+0000] {standard_task_runner.py:63} INFO - Started process 629 to run task
[2025-08-28T06:50:17.381+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_pipeline', 'load_data', 'scheduled__2025-08-27T00:00:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_pipeline_dag.py', '--cfg-path', '/tmp/tmpxb1ml46q']
[2025-08-28T06:50:17.424+0000] {standard_task_runner.py:91} INFO - Job 71: Subtask load_data
[2025-08-28T06:50:17.690+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-28T06:50:17.927+0000] {task_command.py:426} INFO - Running <TaskInstance: ecommerce_pipeline.load_data scheduled__2025-08-27T00:00:00+00:00 [running]> on host 9b893e28b5b2
[2025-08-28T06:50:18.197+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='arel' AIRFLOW_CTX_DAG_ID='ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-27T00:00:00+00:00'
[2025-08-28T06:50:18.199+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-28T06:52:23.405+0000] {logging_mixin.py:188} INFO - ✅ Berhasil load data dari: /opt/***/datasets/online_retail.xlsx
[2025-08-28T06:52:23.405+0000] {logging_mixin.py:188} INFO - ==================================================
[2025-08-28T06:52:23.406+0000] {logging_mixin.py:188} INFO - DEBUG: DATA STRUCTURE
[2025-08-28T06:52:23.407+0000] {logging_mixin.py:188} INFO - ==================================================
[2025-08-28T06:52:23.407+0000] {logging_mixin.py:188} INFO - Columns in DataFrame: ['InvoiceNo', 'StockCode', 'Description', 'Quantity', 'InvoiceDate', 'UnitPrice', 'CustomerID', 'Country']
[2025-08-28T06:52:23.408+0000] {logging_mixin.py:188} INFO - Number of rows: 541909
[2025-08-28T06:52:23.408+0000] {logging_mixin.py:188} INFO - 
First 5 rows:
[2025-08-28T06:52:23.641+0000] {logging_mixin.py:188} INFO -   InvoiceNo StockCode  ... CustomerID         Country
0    536365    85123A  ...    17850.0  United Kingdom
1    536365     71053  ...    17850.0  United Kingdom
2    536365    84406B  ...    17850.0  United Kingdom
3    536365    84029G  ...    17850.0  United Kingdom
4    536365    84029E  ...    17850.0  United Kingdom

[5 rows x 8 columns]
[2025-08-28T06:52:23.643+0000] {logging_mixin.py:188} INFO - 
Data types:
[2025-08-28T06:52:23.646+0000] {logging_mixin.py:188} INFO - InvoiceNo              object
StockCode              object
Description            object
Quantity                int64
InvoiceDate    datetime64[ns]
UnitPrice             float64
CustomerID            float64
Country                object
dtype: object
[2025-08-28T06:52:23.647+0000] {logging_mixin.py:188} INFO - ==================================================
[2025-08-28T06:52:28.878+0000] {logging_mixin.py:188} INFO - ✅ Data berhasil disimpan ke: /tmp/orders.csv
[2025-08-28T06:52:31.118+0000] {logging_mixin.py:188} INFO - [Postgres] Error: column "invoice_no" of relation "orders" does not exist
LINE 2:                 (invoice_no, stock_code, description, quanti...
                         ^
[2025-08-28T06:52:34.879+0000] {logging_mixin.py:188} INFO - [MinIO] File 'orders.csv' berhasil di-upload ✅
[2025-08-28T06:52:34.895+0000] {logging_mixin.py:188} INFO - ✅ ETL process completed successfully!
[2025-08-28T06:52:34.976+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-28T06:52:34.990+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-28T06:52:35.012+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ecommerce_pipeline, task_id=load_data, run_id=scheduled__2025-08-27T00:00:00+00:00, execution_date=20250827T000000, start_date=20250828T065017, end_date=20250828T065235
[2025-08-28T06:52:35.357+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-28T06:52:35.918+0000] {taskinstance.py:3503} INFO - 3 downstream tasks scheduled from follow-on schedule check
[2025-08-28T06:52:36.242+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
